e2e_test/source_inline/commands.toml

```diff
@@ -0,0 +1,108 @@
+# This file contains commands used by the tests.
```

Comment:
The commands added here are actually not used in the `slt` file (although in theory they can). They are now mainly helper commands to run manually.